node/service.go

```diff
@@ -37,6 +37,10 @@ type ServiceContext struct {
 	AccountManager *accounts.Manager
 }
 
+func NewServiceContext(conf *Config, srv map[reflect.Type]Service, mux *event.TypeMux, am *accounts.Manager) *ServiceContext {
```

Comment:
How about changing for using `NewServiceContext` in the main code instead of `&ServiceContext{..}` for consistency?

Reply:
I adopted your suggestion. Thank you.